tools/tdbg/tdbgtest/output_parsing_test.go

```diff
@@ -85,7 +86,9 @@ func TestParseDLQMessages(t *testing.T) {
 		params.ClientFactory = client
 		params.Writer = &b
 	})
-	err = app.Run([]string{
+	ctx, cancel := context.WithTimeout(context.Background(), 15*time.Second)
```

Comment:
does this limit the command timeout to 15 sec?
Sometimes I use `--ct 60` to paginate through super long workflow history or during incident when services are slow.

Reply:
Yes, it limits the command timeout to 15 seconds. I do this because I want the test to fail early

Reply:
oh sorry I mis-read the PR. the 15sec timeout only applies to the test.